UseDefinitions.h

```diff
@@ -24,6 +24,7 @@
 //
 //==============================================================================
 
+#ifndef NO_LEGACY_BEHAVIOR
```

Comment:
I feel this file should ultimately be removed in favor of a CMake file that defines all definitions for all platforms. This way the include paths/order won't matter and behavior will be more consistent

Reply:
@UebelAndre The file will may be auto generated from cmake options as needed. Still under review if it can be fully removed.

Reply:
@UebelAndre  This file is removed